### DISCUSSION TITLE
quarkus cognito user pool extension proxy config

### DISCUSSION THREAD

**Discussion by maghouston** [quarkusio/quarkus#40538] [Community]
we are trying to use quakus cognito userpool extension with a proxy. As mentioned in the documentation, we configure the proxy in the application.yaml as follow:

<img width="407" alt="image" src="https://github.com/quarkusio/quarkus/assets/13933941/1d9100a6-7d95-4246-a49f-05dd95151fa2">

when running the app, we get following error:

`java.base/java.lang.Thread.run(Thread.java:829)\n","errorType":"software.amazon.awssdk.services.cognitoidentityprovider.model.CognitoIdentityProviderException","**errorMessage":"Service returned HTTP status code 407 (Service: CognitoIdentityProvider, Status Code: 407, Request ID: null)**"}
{"timestamp":"2024-05-06T10:08:51.137Z","sequence":865,"loggerClassName":"org.jboss.logging.Logger"
`
He also edited JAVA_OPTS with proxyHost, proxyUser, proxyPassword and proxyPort. we still get the same issue

Could someone help us please. Thanks in advance

**Comment by maxandersen** [comment]
I unfortunately don't know much about cognito user pools, maybe @scrocquesel has an idea?

**Reply by scrocquesel** [reply]
The user open an issue on the extension https://github.com/quarkiverse/quarkus-amazon-services/issues/1261. I expect it to be an NTLM proxy authentication issue with the apache httpclient used underneath.
This could be closed.
